bochs/bx_debug/debug.h

```diff
@@ -231,6 +234,16 @@ void bx_dbg_exit(int code);
 #define bx_dbg_extensions(command) 0
 #endif
 
+
+#define MBP_CX 0x2
+#define MBP_DX 0x4
+#define MBP_BX 0x8
+#define MBP_SP 0x10
+#define MBP_BP 0x20
+#define MBP_SI 0x40
+#define MBP_DI 0x80
```

Comment:
and here (1<<BX_16BIT_REG_DI)
it will be same value
and then drop the #define away and use it 1<<enum as is.